tests/testthat/test-fit_curves.R

```diff
@@ -252,24 +252,24 @@ test_that("predict_conc_from_efficacy works as expected", {
 ###################
 
 test_that(".setup_metric_output works as expected", {
-  obs <- gDRutils:::.setup_metric_output()
+  obs <- .setup_metric_output()
```

Comment:
Just out of curiosity - how you decide which function will not have or will have "dot" before name?
(why `. set_constant_fit_params` -> `set_constant_fit_params` but `.setup_metric_output()`  and `.set_mean_params `stay with dot?